src/utils/DialogIds.ts

```diff
@@ -12,6 +12,7 @@ export const DialogIds = {
     HelloDialogId: "HelloDialog",
     HelpDialogId: "HelpDialog",
     HeroCardDialogId: "HeroCardDialog",
+    PopupSignInDialogId: "PopupSignInDailog",
```

Comment:
Typo: PopupSignInDialog (not Dailog)

Reply:
Done.

Reply:
Need to fix the other 'Dailog' in this line.

Reply:
Done.